content/en/account_management/rbac/log_management.md

```diff
@@ -36,7 +36,7 @@ The following permissions can be granted to manage read access on subsets of log
 
 The following permissions can be granted to manage write access on various log-related account assets:
 
-* **logs_modify_indexes**: Grants a role the ability to modify log indexes. This includes setting inclusion filters for which logs should be routed into an index, limiting which roles have read access on that index (logs_read_index_data), and which roles can modify exclusion filters for that index (logs_write_exclusion_filters). This permission can be granted or revoked from a role via [the Role API][3].
+* **logs_modify_indexes**: Grants a role the ability to modify log indexes. This includes setting inclusion filters for which logs should be routed into an index, limiting which roles have read access on that index (logs_read_index_data), and which roles can modify exclusion filters for that index (logs_write_exclusion_filters). This permission can be granted or revoked from a role via [the Role API][3]. **Note:** This permission also  grants read access on all log indexes and write permissions on all index exclusion filters, since any role that has the ability to modify indexes also has the ability to grant itself these additional permissions. 
```

Comment:
```suggestion
* **logs_modify_indexes**: Grants a role the ability to modify log indexes. This includes setting inclusion filters for which logs should be routed into an index, limiting which roles have read access on that index (logs_read_index_data), and which roles can modify exclusion filters for that index (logs_write_exclusion_filters). This permission can be granted or revoked from a role via [the Role API][3]. **Note:** This permission also grants read access on all log indexes and write permissions on all index exclusion filters, since any role that can modify indexes also can grant itself these additional permissions. 
```